src/Raven.Client/Documents/Operations/TimeSeries/GetTimeSeriesOperation.cs

```diff
@@ -62,12 +59,27 @@ public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, Serve
                     .Append("id=")
                     .Append(Uri.EscapeDataString(_docId))
                     .Append("&name=")
-                    .Append(Uri.EscapeDataString(_timeseries))
-                    .Append("&from=")
-                    .Append(_from.ToString("o"))
-                    .Append("&to=")
-                    .Append(_to.ToString("o"));
-                
+                    .Append(Uri.EscapeDataString(_timeseries));
+
+                if (_ranges != null)
+                {
+                    foreach (var range in _ranges)
+                    {
+                        var f = range.From.Kind == DateTimeKind.Local
+                            ? DateTime.SpecifyKind(range.From, DateTimeKind.Unspecified)
+                            : range.From;
+
+                        var t = range.To.Kind == DateTimeKind.Local
+                            ? DateTime.SpecifyKind(range.To, DateTimeKind.Unspecified)
+                            : range.To;
+
+                        pathBuilder.Append("&from=")
```

Comment:
I don't think this would work properly with the ranges.
I don't believe that there is a good way to correlate multiple `from` / `to` parameters?